px4_offboard/visualizer.py

```diff
@@ -129,6 +135,18 @@ def vehicle_attitude_callback(self, msg):
         self.vehicle_attitude[3] = -msg.q[3]
 
     def vehicle_local_position_callback(self, msg):
+        path_clearing_timeout_ms = (
+            self.get_parameter("path_clearing_timeout_ms")
+            .get_parameter_value()
+            .integer_value
```

Comment:
Could you set the time to seconds, and use a float value?

Reply:
Sure, doing it now. I am also renaming all occurrences of `path_clearing_timeout_ms` to `path_clearing_timeout` effectively dropping unit as seconds are default SI units for time, is that fine? 

Also, would you like me to prepare couple of lines into README providing basic description and instructing users on how to use this feature?

Reply:
@rosiakpiotr That would be awesome, thanks!